src/vs/workbench/services/search/test/common/replace.test.ts

```diff
@@ -105,10 +106,6 @@ suite('Replace Pattern test', () => {
 		actual = testObject.getReplaceString('bla');
 		assert.strictEqual(actual, 'hello');
 
-		testObject = new ReplacePattern('hello$0', { pattern: '(bla)', isRegExp: true });
```

Comment:
Shouldn't this still pass with `$0`? Or is it just removed because there isn't special behavior for `$0` now?

Reply:
$0 was translating to $& prior to the updates made here.  Since that is not happening, this will no longer deliver the results that the tests expect.

Reply:
But are we changing the behavior that the user sees for `$0`? I don't think we should. And if not, then we might as well leave a unit test that still passes.

Reply:
Ya maybe